pkg/servicetest/testdata/services/docker-compose.memcached.yaml

```diff
@@ -1,20 +1,30 @@
-# ddev memcached recipe file
+# DDev memcached recipe file.
 #
-# To use this in your own project: Copy this file to your project's .ddev folder.
-# Edit the 'command' settings to change CLI flags sent to memcached.
-# Defaults to '-m 128' to allocate 128 MB of memcached storage.
```

Comment:
I would like to keep this comment somewhere so it's obvious what the default memory allocation is. Could be down below with the actual arguments if that makes more sense.